### DISCUSSION TITLE
Light slider button for temperature and color

### DISCUSSION THREAD

**Discussion by joezappie** [Clooos/Bubble-Card#784] [Feature requests]
Not sure if this is currently possible but I cant find any info on it. I currently use mushroom, specifically their light card.

It has the option to change Brightness, Color and Temperature all from a single slider by clicking on buttons in the card which is very sleek:
![image](https://github.com/user-attachments/assets/cbbd6c9b-9870-4a21-adb6-071634650cbd)

Clicking the temp button it changes the slider to a kelvin slider:
![image](https://github.com/user-attachments/assets/e7d9f115-2c38-42ac-8312-6c867a9612b5)

Clicking the color button changes it to a gradient slider:
![image](https://github.com/user-attachments/assets/ad1bc70d-be9b-47e4-92b4-c44a48bf26d5)

This is very handy and I use the temperature slider often enough to miss it. I know I can click the icon to bring up the light popup, but this adds another click with a lot more distance between clicks making it much slower to do.

Any chance this is already supported, or something you'd like to support in the future?

**Comment by Clooos** [comment]
Hi, there is no way to do this currently, but I'm planning to make a slider sub-button, similar to the volume slider in the media player card.

![image](https://github.com/user-attachments/assets/1755a2d2-8d45-4e5d-ac9c-77f103bf690d)

I just don't know when I will add this possibility, my to-do list is huge and I'm working on this project mostly by myself.

**Reply by cokeiill** [reply]
Hi, this idea is great 👍， I am very much looking forward to the addition of this new feature. I have researched many methods to adjust the color temperature of bubble card, but it seems that many AI systems cannot achieve it. Will this new feature be added in version 3.0? Looking forward to your reply

**Reply by Clooos** [reply]
This is still not planned for v3.0.0, but this is one of my priorities after its stable release.

**Reply by mateusfn98** [reply]
Up!!!! I was coming here to ask this, would be amazing!

**Reply by joseska** [reply]
+1

**Comment by mateusfn98** [comment]
Someone could help us with that feature in module store ? 🙏🏻
